Realm/Realm/Schema/ObjectSchema.cs

```diff
@@ -175,7 +175,7 @@ public ObjectSchema Build()
                 if (Count == 0)
                 {
                     throw new InvalidOperationException(
-                        $"No properties in {Name}, has linker stripped it? See https://thereIsntSuchDocYet");
+                        $"No properties in {Name}, has linker stripped it? See https://docs.mongodb.com/realm-legacy/docs/dotnet/latest/#linker-stripped-schema");
```

Comment:
This should probably be https://docs.mongodb.com/realm/sdk/dotnet/troubleshooting/#resolve-a--no-properties-in-class--exception